docs/news/index.md

```diff
@@ -1,29 +1,52 @@
+
+| [**Polkadot's Plan for Governing a Blockchain of Blockchains**](https://www.coindesk.com/polkadots-radical-plan-governing-blockchain-blockchains/) | [**SportsDAO**](https://medium.com/@rzurrer/sportsdao-a-self-governing-meritocratic-decentralized-autonomous-organization-that-leverages-2dffac175b52) | [**Pilot Experiment of the Space Decentral Collaboration Process**](https://blog.space.coop/pilot-experiment-of-the-space-decentral-collaboration-process-8d2b9bfe4aa2)
+:-----------:|:-----------:|:-----------:|  
+| [Governance](governance.md) | [DAOs](daos.md) | [DAOs](daos.md)
+| [<img src="https://media.coindesk.com/uploads/2018/03/buttons-860x430.jpg">](https://www.coindesk.com/polkadots-radical-plan-governing-blockchain-blockchains/) | [<img src="https://cdn-images-1.medium.com/max/2000/1*_7m8im8h2WmfWYOGs0xSPA.jpeg">](https://medium.com/@rzurrer/sportsdao-a-self-governing-meritocratic-decentralized-autonomous-organization-that-leverages-2dffac175b52) | [<img src="https://cdn-images-1.medium.com/max/2000/1*D7j0jtilNSqxHxpsxFLeDg.png">](https://blog.space.coop/pilot-experiment-of-the-space-decentral-collaboration-process-8d2b9bfe4aa2)
+| _Author [Rachel Rose O'Leary](https://www.coindesk.com/author/rachelroseoleary/) on Mar 22_ | _Author [Ryan Zurrer](https://medium.com/@rzurrer) on Mar 22_ | _Author [Suzi Bianco](https://medium.com/@Suzi_Bianco) on Mar 26_ 
+| **Who has the authority to change a public blockchain?** It's a question that has been in the minds of top cryptocurrency developers as the many available networks struggle to serve their diverse, often conflicting stakeholders. But that's not to say there aren't norms and best practices – the ability to make and enforce software changes is generally split between the developers that write the code and the computers, or nodes, that install it. | A self-governing, meritocratic, decentralized autonomous organization that leverages the wisdom of the crowd and cryptoeconomic incentives to improve the decisions of a sports franchise.| We’re happy to see people discussing new and exciting concepts for space missions in the Space Decentral forum. For those who had the chance to view our talk at the Ethereum Community Conference, you got a preview at how we propose to operate Space Decentral as an autonomous organization that uses Aragon, in such a way that global citizens can unite to collectively develop an open source and strategic space program.
+| [Read More](https://www.coindesk.com/polkadots-radical-plan-governing-blockchain-blockchains/) | [Read More](https://medium.com/@rzurrer/sportsdao-a-self-governing-meritocratic-decentralized-autonomous-organization-that-leverages-2dffac175b52) | [Read More](https://blog.space.coop/pilot-experiment-of-the-space-decentral-collaboration-process-8d2b9bfe4aa2)
+
+| [**2044 a world governed by DAOs**](http://yermoo.net/2044-a-world-governed-by-daos/) | [**DAOs and the Future of Work**](https://hackernoon.com/daos-and-the-future-of-work-97b4c076f288)
```

Comment:
A small note, this should have the third (empty) column as well, since now with just the two, it makes them proportionally bigger than the other ones.